packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -12,6 +12,89 @@ import 'package:flutter/painting.dart';
 import '../rendering/mock_canvas.dart';
 
 void main() {
+  test('SliderThemeData copyWith, ==, hashCode basics', () {
+    expect(const SliderThemeData(), const SliderThemeData().copyWith());
+    expect(const SliderThemeData().hashCode, const SliderThemeData().copyWith().hashCode);
+  });
+
+  testWidgets('Default SliderThemeData debugFillProperties', (WidgetTester tester) async {
+    final DiagnosticPropertiesBuilder builder = DiagnosticPropertiesBuilder();
+    const SliderThemeData().debugFillProperties(builder);
+
+    final List<String> description = builder.properties
+        .where((DiagnosticsNode node) => !node.isFiltered(DiagnosticLevel.info))
+        .map((DiagnosticsNode node) => node.toString())
+        .toList();
+
+    expect(description, <String>[]);
+  });
+
+
+  testWidgets('SliderThemeData implements debugFillProperties', (WidgetTester tester) async {
+    final DiagnosticPropertiesBuilder builder = DiagnosticPropertiesBuilder();
+    const SliderThemeData(
+      trackHeight: 7.0,
+      activeTrackColor: Color(0xFF000001),
+      inactiveTrackColor: Color(0xFF000002),
+      disabledActiveTrackColor: Color(0xFF000003),
+      disabledInactiveTrackColor: Color(0xFF000004),
+      activeTickMarkColor: Color(0xFF000005),
+      inactiveTickMarkColor: Color(0xFF000006),
+      disabledActiveTickMarkColor: Color(0xFF000007),
+      disabledInactiveTickMarkColor: Color(0xFF000008),
+      thumbColor: Color(0xFF000009),
+      overlappingShapeStrokeColor: Color(0xFF000010),
+      disabledThumbColor: Color(0xFF000011),
+      overlayColor: Color(0xFF000012),
+      valueIndicatorColor: Color(0xFF000013),
+      overlayShape: RoundSliderOverlayShape(),
+      tickMarkShape: RoundSliderTickMarkShape(),
+      thumbShape: RoundSliderThumbShape(),
+      trackShape: RoundedRectSliderTrackShape(),
+      valueIndicatorShape: PaddleSliderValueIndicatorShape(),
+      rangeTickMarkShape: RoundRangeSliderTickMarkShape(),
+      rangeThumbShape: RoundRangeSliderThumbShape(),
+      rangeTrackShape: RoundedRectRangeSliderTrackShape(),
+      rangeValueIndicatorShape: PaddleRangeSliderValueIndicatorShape(),
+      showValueIndicator: ShowValueIndicator.always,
+      valueIndicatorTextStyle: TextStyle(color: Colors.black),
+    ).debugFillProperties(builder);
+
+    final List<String> description = builder.properties
+        .where((DiagnosticsNode node) => !node.isFiltered(DiagnosticLevel.info))
+        .map((DiagnosticsNode node) => node.toString())
+        .toList();
+
+    expect(description, <String>[
+      'trackHeight: 7.0',
+      'activeTrackColor: Color(0xff000001)',
+      'activeTrackColor: Color(0xff000001)',
```

Comment:
I am assuming it's not intentional that `activeTrackColor` is printed twice? I am removing it in https://github.com/flutter/flutter/pull/34012.